outputs.tf

```diff
@@ -1,23 +1,45 @@
 output "fargate_service" {
-  value = module.fargate.ecs_service
+  value = aws_ecs_service.service
 }
 
 output "fargate_service_security_group" {
-  value = module.fargate.service_sg
+  value = aws_security_group.fargate_service_sg
 }
 
-output "codedeploy_appspec_json" {
-  value = module.fargate.codedeploy_appspec_json
+output "task_definition" {
+  value = aws_ecs_task_definition.task_def
+}
+
+output "codedeploy_deployment_group" {
+  value = aws_codedeploy_deployment_group.deploymentgroup
+}
+
+output "codedeploy_basic_appspec_json" {
+  value = jsonencode({
+    version = 1
+    Resources = [{
+      TargetService = {
+        Type = "AWS::ECS::SERVICE"
+        Properties = {
+          TaskDefinition = aws_ecs_task_definition.task_def.arn
+          LoadBalancerInfo = {
+            ContainerName = local.container_definitions[0].name
+            ContainerPort = var.image_port
+          }
+        }
+      }
+    }]
+  })
 }
```

Comment:
Why "basic"? Is the intent that the root module would catch this output and edit it before re-outputing it? If so, do we want to skip the `jsonencode` and make the root module do that? Or is that weird to make the root module take on that responsibility?

I don't think we expect a ton of customization. Just hooks. Seems like it would be pretty easy to bring those in as a variable and build them in here.

Reply:
Good point, but those hooks' lambda functions that it would execute are not created by the standard fargate module, so I didn't really think it belonged here. But you're right, it wouldn't be hard to add the hooks. Want me to change the output to remove the word "basic" in preparation?

Reply:
Done